Source/Blazorise/Licensing/BlazoriseLicenseProvider.cs

```diff
@@ -95,14 +127,182 @@ private async void BackgroundWorker_DoWork( object sender, DoWorkEventArgs e )
         }
     }
 
+    private static BlazoriseLicenseResult ResolveBlazoriseLicenseResult( License license )
+    {
+        if ( license is null )
+            return BlazoriseLicenseResult.Unlicensed;
+
+        if ( license.Properties.TryGetValue( Constants.Properties.LICENSE_TYPE, out var licenseType ) && Enum.TryParse<BlazoriseLicenseType>( licenseType, true, out var licenseTypeAsEnum ) )
+        {
+            switch ( licenseTypeAsEnum )
+            {
+                case BlazoriseLicenseType.Community:
+                    return BlazoriseLicenseResult.Community;
+                case BlazoriseLicenseType.Regular:
+                    return BlazoriseLicenseResult.Licensed;
+                case BlazoriseLicenseType.Trial:
+                    return BlazoriseLicenseResult.Trial;
+                default:
+                    break;
+            }
+        }
+
+        return BlazoriseLicenseResult.Licensed;
+    }
+
+    internal int? GetDataGridRowsLimit()
```

Comment:
All the functions seems to do the sam thing. Can we make an additional function to resolve the with the same logic?

eg. `internal int? GetRowsLimit(string limitPropertyName, int communityMaxRows, int unlicensesMaxRows)`